.travis.yml

```diff
@@ -1,28 +1,54 @@
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: osx
+      osx_image: xcode7.3
+      compiler: clang
+
 language: cpp
 
-compiler:
-  - gcc
+cache:
+  directories:
+    - ${HOME}/local
+    - ${HOME}/cpanm
 
 addons:
   hosts:
     - 127.0.0.1.xip.io
     - alternate.127.0.0.1.xip.io
 
 before_install:
-  # upgrade g++ and libstdc++ to build nghttp2
-  - sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
-  - sudo apt-add-repository --yes ppa:smspillaz/cmake-2.8.12
-  - sudo apt-get --yes update
-  - sudo apt-get install --yes cmake cmake-data g++-4.8 libstdc++-4.8-dev php5-cgi wget
-  - if [ "$CXX" = "g++" ]; then export CXX="g++-4.8"; fi
-  - $CXX --version
-  # for speed, pre-install deps installed in `before_script` section as ubuntu packages
-  - sudo apt-get install -qq cpanminus libipc-signal-perl liblist-moreutils-perl libwww-perl libio-socket-ssl-perl zlib1g-dev
+  - source ./misc/travis/export_env
```

Comment:
Or, is there any reason to have the environment variables listed in a dedicated file considering the fact that `export_env` is only used once?